scripts/gen-dumbfs.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+##############################################################################
+##
+##  This file is part of the Chaos Kernel, and is made available under
+##  the terms of the GNU General Public License version 2.
+##
+##  Copyright (C) 2017 - Jakob Kellendonk <jakob.kellendonk@epitech.eu>
+##
+##############################################################################
+
+if (( $# != 2 ))
+then
+	echo -E "usage: ./${0} IN_DIR OUT_FILE"
```

Comment:
Why `-E` ?

Btw `echo` is not always present, you should rely on `printf` instead.

Reply:
echo is a bash built-in, so it's probably safe to say it's always present.
-E is to avoid interpretation of the string.

Reply:
Flags depends on the version of bash. I remember having troubles when using `echo -e` and some versions of debian.